TestCases/compliance-level-3/0072-feel-in/0072-feel-in-test-01.xml

```diff
@@ -0,0 +1,2735 @@
+            <expected>
+                <value xsi:type="xsd:boolean">true</value>
+            </expected>
+        </resultNode>
+    </testCase>
+
+    <testCase id="number_005_1">
+        <description>number in: e1 in >=e2</description>
+        <resultNode name="number_005_1" type="decision">
+            <expected>
+                <value xsi:type="xsd:boolean">true</value>
+            </expected>
+        </resultNode>
+    </testCase>
+
+    <testCase id="number_005_a">
+        <description>number not in: e1 in >=e2</description>
+        <resultNode name="number_005_a" type="decision">
+            <expected>
+                <value xsi:type="xsd:boolean">false</value>
+            </expected>
+        </resultNode>
+    </testCase>
+
+    <testCase id="number_006">
+        <description>number in: e1 in >e2</description>
+        <resultNode name="number_006" type="decision">
+            <expected>
+                <value xsi:type="xsd:boolean">true</value>
+            </expected>
+        </resultNode>
+    </testCase>
+
+    <testCase id="number_006_a">
+        <description>number not in: e1 in >e2</description>
+        <resultNode name="number_006_a" type="decision">
+            <expected>
+                <value xsi:type="xsd:boolean">false</value>
+            </expected>
+        </resultNode>
+    </testCase>
+
+    <testCase id="number_007_a">
+        <description>number in: e1 in (e2..e3)</description>
+        <resultNode name="number_007_a" type="decision">
+            <expected>
+                <value xsi:type="xsd:boolean">false</value>
+            </expected>
+        </resultNode>
+    </testCase>
+
+    <testCase id="number_007_b">
+        <description>number in: e1 in (e2..e3)</description>
+        <resultNode name="number_007_b" type="decision">
+            <expected>
+                <value xsi:type="xsd:boolean">false</value>
+            </expected>
+        </resultNode>
+    </testCase>
+
+    <testCase id="number_007_c">
+        <description>number in: e1 in (e2..e3)</description>
+        <resultNode name="number_007_c" type="decision">
+            <expected>
+                <value xsi:type="xsd:boolean">true</value>
+            </expected>
+        </resultNode>
+    </testCase>
+
+    <testCase id="number_007_d">
+        <description>number in: e1 in (e2..e3)</description>
+        <resultNode name="number_007_d" type="decision">
+            <expected>
+                <value xsi:type="xsd:boolean">false</value>
+            </expected>
+        </resultNode>
+    </testCase>
+
+    <testCase id="number_007_e">
+        <description>number in: e1 in (e2..e3)</description>
+        <resultNode name="number_007_e" type="decision">
+            <expected>
+                <value xsi:type="xsd:boolean">false</value>
+            </expected>
+        </resultNode>
+    </testCase>
+
+    <testCase id="number_008_a">
+        <description>number in: e1 in (e2..e3]</description>
```

Comment:
I believe typo, 
either the form `number not in:` shall be used,
either the simpler form `number:` would avoid misunderstanding as checking this description